src/FlatSelect/components/SelectSearchRow/SearchRow.tsx

```diff
@@ -117,7 +117,7 @@ export const SelectSearchRow: React.FC<SelectSearchRowProps> = ({
 			>
 				{text}
 			</div>
-			<Сhildren />
+			<RenderСhildren />
```

Comment:
Это всё-таки не компонент, не класс, это просто функция внутри компонента, так будет понятнее
```suggestion
			{renderChildren()}
```
Либо действительно сделать это отдельным компонентом